lib/mm_network.pm

```diff
@@ -49,7 +49,8 @@ sub get_host_resolv_conf {
 
 sub is_networkmanager {
     my $is_nm = (script_run('readlink /etc/systemd/system/network.service | grep NetworkManager') == 0);
-    record_info('NetworkManager', (($is_nm) ? 'NetworkManager has been detected.' : 'NetworkManager has not been detected.'));
+    my $msg = $is_nm ? 'NetworkManager has been detected.' : 'NetworkManager has not been detected.';
+    record_info('NetworkManager', "$msg");
```

Comment:
I don't see the improvement here

Reply:
This is only to prevent the uninitialized value warnings in the autoinst-logs